pages/videos.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
-    <VideoTable :videos="publishedVideos" :headers="headers" :itemsPerPage="1000" class="hidden-xs-only" />
-    <VideoTable :videos="publishedVideos" :headers="mobileHeaders" :itemsPerPage="1000" :showExpand="false" class="hidden-sm-and-up" />
+    <VideoTable ref="desktopTable" @query-changed="handleQueryChange" :videos="publishedVideos" :headers="headers" :itemsPerPage="1000" class="hidden-xs-only" />
```

Comment:
Okay, this feeds into my hunch of how VideoTable needs to be refactored.

`VideoTable` should be turned into just the v-data-table and associated javascript.  Then move the search input and the filtering onto this page (so the search input is right there with the query param, where it wants to be) and call the `VideoTable` component twice.

Does that make sense?

Reply:
Wait, ignore this.